### PR TITLE
Fix sample app problem in RouteServiceProvider

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Hello Viktor! 👋 
Your PR is excellent 💎

Thank you!!

**Comment by canvural** [comment]
Hi,

Can you send this to `master` branch also?

**Comment by szepeviktor** [comment]
> Can you send this to `master` branch also?

Done. 🍏 
https://github.com/nunomaduro/larastan/commits/master

**Comment by szepeviktor** [comment]
@canvural We have two somethings on `master`
Please see https://github.com/nunomaduro/larastan/actions/runs/1725378835

**Comment by canvural** [comment]
Yes, it'll be fixed when `l9` branch gets merged 👍🏽

File: tests/laravel-test.sh [hunk 1]
```diff
@@ -18,8 +18,11 @@ sed -e 's|@return \\Illuminate\\Http\\Response$|@return \\Symfony\\Component\\Ht
 
 echo "Fix https://github.com/nunomaduro/larastan/pull/378#issuecomment-565706907"
 sed -e 's/string/string|void/' -i app/Http/Middleware/Authenticate.php
-sed '0,/}/s/}/}\nreturn;/' -i app/Http/Middleware/Authenticate.php
+sed -e '0,/}/s/}/}\nreturn;/' -i app/Http/Middleware/Authenticate.php
 
+echo "Fix commented namespae property in RouteServiceProvider"
```
Comment (williamdes) [inline tests/laravel-test.sh:23]: typo: namespace

Comment (szepeviktor) [reply tests/laravel-test.sh:23]: Thank you.

Comment (szepeviktor) [reply tests/laravel-test.sh:23]: Fixed in master https://github.com/nunomaduro/larastan/pull/1107/files

**Pull request merged**